ta/arch/arm/link.mk

```diff
@@ -25,7 +25,7 @@ reverse = $(if $(wordlist 2,2,$(1)),$(call reverse,$(wordlist 2,$(words $(1)),$(
 link-ldadd  = $(LDADD)
 link-ldadd += $(addprefix -L,$(libdirs))
 link-ldadd += $(addprefix -l,$(call reverse,$(libnames)))
-ldargs-$(binary).elf := $(link-ldflags) $(objs) $(link-ldadd) $(libgcc$(sm))
```

Comment:
As far as I know we're building OK with this today, what's changed? I don't mind removing it, but we should know why we're doing it and why we don't need it any longer. I guess the answer is that `libutils` takes care of everything now (and will continue to do so if the compiler ever needs something new).